primefaces/src/main/java/org/primefaces/renderkit/HeadRenderer.java

```diff
@@ -126,8 +129,16 @@ public void encodeBegin(FacesContext context, UIComponent component) throws IOEx
         }
 
         if (applicationContext.getConfig().isClientSideLocalizationEnabled()) {
-            Locale locale = LocaleUtils.getCurrentLocale(context);
-            encodeJS(context, LIBRARY, "locales/locale-" + locale.getLanguage() + ".js");
+            try {
+                Locale locale = LocaleUtils.getCurrentLocale(context);
+                encodeJS(context, LIBRARY, "locales/locale-" + locale.getLanguage() + ".js");
+            }
+            catch (FacesException e) {
+                if (LOGGER.isLoggable(Level.WARNING)) {
+                    LOGGER.log(Level.WARNING,
+                            "Failed to load client side locale.js. {0}", e.getMessage());
```

Comment:
I would log the filename inclusive language

Reply:
Good idea